### ISSUE TITLE
Accessory and Component models consistency

### ISSUE THREAD

**Issue by TerraTech** [grokability/snipe-it#9647]
**Server (please complete the following information):**
 - Snipe-IT Version: v5.1.5
 - OS: Alpine Linux v3.13
 - Web Server: Apache2
 - PHP Version: PHP-7

**Describe the solution you'd like**
Bring consistency to the Accessory and Component models.

In a nutshell:
Accessories are checked out to people.
Components are checked out to assets.

I had just started with Snipe-IT and I accidently added a SDD drive to Accessories thinking I could assign it to a desktop asset.  Once I found it didn't work that way, I then saw that Component was what I was needing.  I created a new Component and started to copy across the Accessory details to the new Component.  That is when I saw the missing fields.

I believe that an Accessory should have a Serial Number .
I believe that a Component should have Manufacturer, Model, and Supplier.

Currently, I'm not sure why there is this disparity between Accessories and Components.  It doesn't make sense to me why they aren't treated equally.

**Describe alternatives you've considered**
There is no current workaround due to missing fields.

**Additional context**
Comparison of Accessory and Component fields:
![Snipe-IT_Accessory_vs_Component](https://user-images.githubusercontent.com/1118433/120089523-1ee47d80-c0b0-11eb-84a7-347baf0de649.jpg)

**Comment by TerraTech** [comment]
I would also like to see a **Notes** field for both.

**Comment by hmacugova** [comment]
Hi, 
+1 for this functionality.
Also will be good, when I can assign accessories or components to users/assets/locations, like Assets. For example Accessories is keyboard/mouse, but I can assign it to laptop (for users), or desktop (for assets).
Components can have one Part Number, but I can have 15 pieces, so more Serial numbers. In components, there are Quantities and minimum qty but I can write only one serial number.
Thank you so much

**Comment by azeem51** [comment]
I agree with TerraTech. I definitely want to see supplier/vendor, manufacturer, model, notes and warranty fields for components. And yes, some accessories do have a serial number.

**Comment by dthompson0627** [comment]
Adding a comment here to hopefully show continued interest in this feature. I'd love to have that functionality.

**Comment by bilias** [comment]
+1

**Comment by smithj33** [comment]
Jumping on this. Component fields need to match Accessory. So weird they haven't fixed this in 3 years.

**Comment by snipe** [comment]
I think most of these problems were sorted a long time ago. Anything that's still; present, please open a new ticket.
